connect/clients/ipfs.py

```diff
@@ -26,61 +27,58 @@ def __init__(self, uri, replication_factor):
         self._get_peers_uri = self._uri + "/peers"
         self._unpin_cid_uri = self._uri + "/pins/"
 
-    async def persist_json_to_ipfs(self, payload_dict):
+    async def persist_json_to_ipfs(self, payload: dict):
```

Comment:
We can use the [Tuple](https://docs.python.org/3/library/typing.html#typing.Tuple) type hint here to annotate the return value.